### ISSUE TITLE
PA Review: User Authentication

### ISSUE THREAD

**Issue by jlamb916** [jlamb916/Striva#2]
- [x] Users can sign up, sign in, log out
- [x] Users can use a demo login to try the site
- [x] Users can't use certain features without logging in (creating routes, viewing stats)
- [x] Styling
- [x] Smooth, bug-free navigation

**Comment by ryan-mapa** [comment]
- [ ] I can't seem to log out (getting a 404)
- [ ] buttons need hover styling
- [ ] need to add a font to your site (serifs are bad.)

**Comment by jlamb916** [comment]
Hey Ryan, I added the hovering effects on buttons and changed the fonts. Hopefully it looks better. 

As for the log out button. I ran into the 404 error a few times and wasn't sure why. I haven't been able to replicate it since it happened, but I changed my code to demo dispatch to a button rather than an input (maybe it was interfering with input submit button?) Anyways, I tested it out for a good amount of time and didn't run into the issue again.
